presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -66,12 +66,25 @@ Mathematical Functions
 
     Returns the value of ``string`` interpreted as a base-``radix`` number.
 
+.. function:: hypergeometric_cdf(populationSize, numberOfSuccesses, sampleSize, observedSuccesses) -> double
+
+    Compute the Hypergeometric cdf with given populationSize, numberOfSuccesses, sampleSize
+    and observed_successes:  P(N <= observedSuccesses). The populationSize must be a positive integer
```

Comment:
Please change `observedSuccesses` to `value` (just to be consistent with the other functions)

Reply:
(please explain in the text here that value is observedSuccesses)